### DISCUSSION TITLE
Radio Paradise Cover Art Problem

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1282] [Fixed Issues]
Anyone else find that RP suddenly stops showing cover art and reverts to the rAudio default? Reselecting the station gets it back until the next time. Metadata info is shown as normal.

**Comment by cmh714** [comment]
just started testing to see for you, and working as it should so far.....using the Rock station.

**Reply by R101G** [reply]
Thanks @cmh714

**Comment by R101G** [comment]
The RP add-on seems to affect other things too. If I go from RP to an album replace and play, it sometimes replaces but does not play. If I go back from the album to RP, the RP metadata does not always appear until the next track change.

Is there an easy way to re-install the RP package?

**Comment by R101G** [comment]
I have not seen this problem in the two days since I last did a pacman update, which might be coincidence. I assume that if for any reason RP does not get the cover art, it displays the rAudio logo rather than the station logo, so this might be due to something in the previous pacman update causing the fetching of cover art to fail.
